### DISCUSSION TITLE
Broken link - Learning Path: Mastering spaCy for Natural Language Processing

### DISCUSSION THREAD

**Discussion by gregomelo** [explosion/spaCy#11842] [Help: Other Questions] [answered]
Hi.

The link **VIEW MORE** in **Learning Path: Mastering spaCy for Natural Language Processing** (https://spacy.io/universe/project/learning-path-spacy) is pointing to (https://www.safaribooksonline.com/library/view/learning-path-mastering/9781491986653/) who lead to a **Not Found** page.

**Comment by polm** [comment, marked as answer]
Thanks for the heads up! We will fix that, or maybe remove it, since it looks like that resource hasn't been updated and it's pretty old at this point.
